src/components/Button/Button.module.scss

```diff
@@ -19,10 +19,14 @@
     color: $white;
     user-select: none;
   }
+
+  @media (max-width: 700px) {
+    width: 100%;
+  }
```

Comment:
Make common breakpoints in [`Properties.scss`](src/styles/Properties.scss)

Previous implementation: https://github.com/GTBitsOfGood/npp/blob/cb4fbe3b04ec7f0a62dde5470b150d7ae6203275/src/styles/Properties.scss
Previous usage: https://github.com/GTBitsOfGood/npp/blob/cb4fbe3b04ec7f0a62dde5470b150d7ae6203275/src/components/Header/Header.module.scss